terraform/modules/google-cloud/apps/relay/main.tf

```diff
@@ -181,7 +147,7 @@ resource "google_compute_instance_template" "application" {
 
   project = var.project_id
 
-  name_prefix = "${local.application_name}-${each.key}-"
+  name_prefix = "${local.application_name}-template-${each.key}-"
```

Comment:
This name is bumped as well to taint the instance groups and cause them to be rolled.

If we don't do this, then GCP complains that the existing instance group's subnet interface is attached to both the old and new VPC networks.